Airwallex/Redirect/AWXRedirectActionProvider.m

```diff
@@ -45,6 +45,19 @@ - (void)handleNextAction:(AWXConfirmPaymentNextAction *)nextAction {
     }
 }
 
+- (void)confirmPaymentIntentWithPaymentMethodName:(NSString *)paymentMethodName additionalInfo:(NSDictionary<NSString *, NSString *> *)additionalInfo {
+    [self confirmPaymentIntentWithPaymentMethodName:paymentMethodName additionalInfo:additionalInfo flow:AWXPaymentMethodFlowApp];
+}
+
+- (void)confirmPaymentIntentWithPaymentMethodName:(NSString *)paymentMethodName
+                                   additionalInfo:(NSDictionary<NSString *, NSString *> *_Nullable)additionalInfo
+                                             flow:(AWXPaymentMethodFlow)flow {
+    AWXPaymentMethod *paymentMethod = [[AWXPaymentMethod alloc] init];
+    paymentMethod.type = paymentMethodName;
+    paymentMethod.additionalParams = additionalInfo;
+    [self confirmPaymentIntentWithPaymentMethod:paymentMethod paymentConsent:nil flow:flow];
```

Comment:
Has this method been OCMVeirfy-ed to see whether it has been executed with right parameters?